src/configuration-parser.ts

```diff
@@ -1,16 +1,18 @@
 import * as Ajv from 'ajv';
 import * as fs from 'fs';
 import * as path from 'path';
+import * as util from 'util';
 
 export class ConfigurationParser {
     private typewizConfig: any;
 
-    public parse(configurationPath: string) {
-        const typewizConfigSchema = JSON.parse(fs.readFileSync('src/typewiz.json', { encoding: 'utf8' }));
+    public async parse(configurationPath: string): Promise<void> {
+        const readFileAsync = util.promisify(fs.readFile);
```

Comment:
You should probably do this outside, promisifying is expensive and we shouldn't do it every time `parse()` is called ;)

Reply:
All I meant was that you move `const readFileAsync = ...` outside the class, so that the promisification only happens once. There shouldn't be any new typing problems (literally cut and paste line 10 to above the `class ...` line.

Reply:
Ah, of course :) I just made the change.